src/BlockTypes/ProductDetails.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * ProductDetails class.
+ */
+class ProductDetails extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-details';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		$tabs = $this->render_tabs();
+
+		$classname = $attributes['className'] ?? '';
+
+		return sprintf(
+			'<div class="wp-block-woocommerce-single-product-details %1$s">
```

Comment:
The styles are now broken due to the latest changes to rename the block from `single-product-details` to `product-details`. The following should do the trick:
```suggestion
			'<div class="wc-block-woocommerce-product-details %1$s">
```
I also suggest renaming the class prefix from `wp-block` to `wc-block` so it is aligned with the standards we are using in other blocks.

Reply:
We decided to avoid adding a custom class. We should use WP generated class (more context: p1675160559571929-slack-C02UBB1EPEF)

Reply:
> We decided to avoid adding a custom class. We should use WP generated class (more context: p1675160559571929-slack-C02UBB1EPEF)

Oh cool, I missed that convo, thanks for sharing! I'll also go ahead and update #8284 with this prefix.

In case someone else comes across this convo: the reason for this decision was that for new blocks, it's a good idea not to introduce new classes, as block themes interact with blocks via theme.json.